nomenclature/cli.py

```diff
@@ -50,6 +50,27 @@ def cli_valid_project(
 ):
     """Assert that `path` is a valid project nomenclature
 
+    Parameters
+    ----------
+    path : Path
+        directory path to the file of interest
+    dimensions : List[str], optional
+        List of dimensions to be checked, default to None which implies that all
+        directories in `dimensions` will be checked
```

Comment:
```suggestion
        directories in `definitions` will be checked
```

Reply:
oupsi I keep inverting dimensions and definitions, sorry for that. thank you @phackstock !!